src/app/screens/swap/swapInfoBlock/index.tsx

```diff
@@ -84,6 +106,23 @@ export function SwapInfoBlock({ swap }: { swap: UseSwap }) {
             <DD>{swap.swapInfo?.lpFee ?? '--'}</DD>
             <DT>{t('ROUTE')}</DT>
             <DD>{swap.swapInfo?.route ?? '--'}</DD>
+            {swap.isServiceRunning && (
+              <>
+                <DT>{t('SPONSOR_TRANSACTION')}</DT>
+                <ToggleContainer>
+                  <CustomSwitch
+                    onColor={theme.colors.purple_main}
+                    offColor={theme.colors.background.elevation3}
+                    onChange={toggleFunction}
+                    checked={swap.isSponsored}
+                    uncheckedIcon={false}
+                    checkedIcon={false}
+                    height={19}
+                    width={36}
```

Comment:
how come there is height/width here as well as in CSS on L16,17?

Reply:
On L16,17 the height and width is set for the react-switch-handle which is the circle within the switch. Here the main switch body dimensions are set. We need to alter the two since according to the design we are not using the default dimensions

Reply:
ok, let's clean this up when we build out the UI revamp